tests/phpunit/tests/rest-api/wpRestTemplatesController.php

```diff
@@ -678,4 +684,39 @@ protected function find_and_normalize_template_by_id( $templates, $id ) {
 		return null;
 	}
 
+	/**
+	 * @ticket 56467
+	 * @covers WP_REST_Templates_Controller::get_template_fallback
+	 */
+	// TODO: check how to make this test work..
+	// https://github.com/WordPress/gutenberg/pull/42520/files#diff-5dcd7f0ba0e053098e93f6907913da73787fb7487e05d37e3249fb1c0ec434d6
```

Comment:
How do you suggest to make these tests for core? In GB we have a different approach: https://github.com/WordPress/gutenberg/pull/42520/files#diff-5dcd7f0ba0e053098e93f6907913da73787fb7487e05d37e3249fb1c0ec434d6.

Should we create a new test theme or update an existing one or something else? 🤔 --cc @spacedmonkey

Reply:
Used 'block-theme' eventually and tested with `slugs`.